src/system/api/rqidApi.ts

```diff
@@ -1,126 +1,466 @@
+import { ArmorDataSource } from "../../data-model/item-data/armorData";
+import { ItemTypeEnum } from "../../data-model/item-data/itemTypes";
+import { SkillDataSource } from "../../data-model/item-data/skillData";
 import { RqgItem } from "../../items/rqgItem";
-import { getGame, localize } from "../util";
-
-/**
- * Return the highest priority item matching the supplied rqid and lang from the items in the World. If not
- * found return the highest priority item matching the supplied rqid and lang from the installed Compendia.
- *
- * Example:
- * ```
- * game.system.api.fromRqid("someid", "es")
- * ```
- */
-export async function fromRqid(
-  rqid: string,
-  lang: string = "en"
-): Promise<RqgItem | undefined> {
-  if (!rqid) {
-    return undefined;
-  }
+import { RQG_CONFIG } from "../config";
+import { getGame, localize, toKebabCase } from "../util";
 
-  const worldItem = await fromWorldRqid(rqid, lang);
+export class Rqid {
+  /**
+   * Return the highest priority Document matching the supplied rqid and lang from the Documents in the World. If not
+   * found return the highest priority Document matching the supplied rqid and lang from the installed Compendia.
+   *
+   * Example:
+   * ```
+   * game.system.api.fromRqid("someid", "es")
```

Comment:
This is not correct anymore I think? Maybe we should just remove the example. Usually not a good idea to have code in comments anyway...

Reply:
I removed the example and started a wiki page where we can document this: https://github.com/sun-dragon-cult/fvtt-system-rqg/wiki/rqid

And I had already corrected the item sheet.